### DISCUSSION TITLE
[✨] fall through attributes

### DISCUSSION THREAD

**Discussion by agufagit** [QwikDev/qwik-evolution#166] [Proposals For Qwik]
### Is your feature request related to a problem?

like v-bind="$attrs" in vue, {...$$restprops} in svelte, and splitprops in solid

right now Qwik shows error when specify attributes not in props type definition

```javascript
interface IButtonProps {
  text: string
}

interface IButtonPropsQwik {
  props: IButtonProps 
}

export default component$(({ props }: IButtonPropsQwik ) => {...})

<Custombutton id="new" props={...} />
```

the error is `Property 'id' does not exist on type 'IntrinsicAttributes & TransformProps<IButtonProps> & ComponentBaseProps & { children?: JSXChildren; }`

### Describe the solution you'd like

attributes defined other than the ones in props are automatically fall through to a variable like `restprops`, and there is a way to bind this `restprops` to a html element

### Describe alternatives you've considered

or do it like splitprops in solid

**Comment by gioboa** [comment]
Hey @agufagit, thanks for opening this.
A lot of things happened during this months, with the latest version of Qwik is it still a valid enhancement?

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
